ci-operator/step-registry/openstack/deprovision/mirror/openstack-deprovision-mirror-commands.sh

```
@@ -40,13 +42,34 @@ if [[ -f ${SHARED_DIR}"/MIRROR_SSH_IP" ]]; then
+        ssh_mirror() { ssh_direct "$@"; }
+        scp_mirror() { scp_direct "$@"; }
+    fi
+
+    ssh_mirror bash - <<EOF
     mkdir -p /tmp/mirror-logs
     sudo podman logs registry > /tmp/mirror-logs/registry.log || true
     sudo chown -R ${BASTION_USER}: /tmp/mirror-logs || true
     tar -czC "/tmp" -f "/tmp/mirror-logs.tar.gz" mirror-logs/
 EOF
-    scp_via_proxy ${BASTION_USER}@${mirror_ip}:/tmp/mirror-logs.tar.gz ${ARTIFACT_DIR}
+    scp_mirror ${BASTION_USER}@${mirror_ip}:/tmp/mirror-logs.tar.gz ${ARTIFACT_DIR}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)